src/Resources/translations/SonataAdminBundle.de.xliff

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This files should be kept. They are needed for this: https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_orm_admin.php#L31